### ISSUE TITLE
Add a link to the github issue tracker to the site

### ISSUE THREAD

**Issue by malemburg** [python/pythondotorg#189]
I don't know where the feedback submissions are sent, but given that they don't end up on the pydotorg-www mailing list and are also not automatically added to this tracker, I think it would be better to either replace the feedback form with a simple link to this tracker or at least provide alternatives for sending feedback:
- submit a ticket on the github tracker (https://github.com/python/pythondotorg/issues/new)
- send email to pydotorg-www to get something fixed
- use the feedback form

**Comment by benjaminp** [comment]
See also #65 .

**Comment by techtonik** [comment]
Note that Github requires yet another password and not everyone get what this site does and how it works.

**Comment by ned-deily** [comment]
Since this issue keeps coming up and it isn't clear whether there should be a better solution and since the `Help & General Contact` link at the bottom of web pages seems to often be overlooked, I've added a `Help` item to the `About` site tree cascade; it takes you to the same https://www.python.org/about/help/ page.  And, on that page, I've added links to this issue tracker for the web site.  Does that seem good enough?
